spec/spec_helper.rb

```diff
@@ -0,0 +1,94 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```

Comment:
How do you feel about removing all the comments from this file? They only add unnecessary noise. We want to be able to work on this file and understand it as fast as possible.